test/index.ts

```diff
@@ -1,5 +1,5 @@
-describe('A basic test', () => {
-  it('to have something pass', () => {
+describe('A basic test', (): void => {
```

Comment:
Is this needed by the linter? Seems overkill to me to mention void.

Reply:
Yea, the `eslint-config-es` config is quite restrictive. Notice that in the `it` function below a dot was added to the end of the sentence, that was also a new requirement.

Reply:
The nice thing about eslint is that the `--fix` command can easily apply/undo such things, so we can still undo this easily in the future.
So let's go forward with this one for now, we can still re-evaluate later on.